packages/forma-36-tokens/README.md

```diff
@@ -35,7 +35,7 @@ import F36Tokens from '@contentful/forma-36-tokens';
 #### SCSS
 
 ```scss
-@import '@contentful/forma-36-tokens/dist/css/index.scss';
+@import '@contentful/forma-36-tokens/dist/css/index.css';
```

Comment:
Or is it supposed to be
```suggestion
@import '@contentful/forma-36-tokens/dist/scss/index.scss';
```

Reply:
The original change I have worked for me.

Reply:
Hey @davidfateh  - sorry for the delay. It should be your last suggestion - `'@contentful/forma-36-tokens/dist/scss/index.scss';` but I believe the output is identical anyway. Perhaps this should be changed so that the scss version exports scss variables rather than CSS custom properties